markdown/chapters/univariate-analysis/main.rmd

```diff
@@ -0,0 +1,1672 @@
+
+Compared to the control scaled plot and based on the information discussed to far, it appears that truncating the y-scale has marginally improved the respondents' ability to judge the height of the bar. This makes sense as truncating the scale results in less of a spread of values on the same sized scale, and so wider gap between each value and thus potentially allows a more accurate judging of values. There is a discrepancy here between languages in terms of the axis labeling, with the R plot being incremented in steps of 10 for both the control and truncated plots and the Python in steps of 5 for the control and steps of 2.5 for the truncated. In the multivariate analysis we will subset based on language, university degree and more to compare between these and assess whether this numbering made a difference, alongside other factor, such as the impact of standard form on the Python responses. 
+
+Now we consider the shape of the distributions of the responses for the control and truncated plots, with the response set for logarithmically-scaled plot excluded from this, since the high magnitude values will stretch out the x-axis and not allow us to properly see the distributions.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding the control and truncated plots"}
+
+brks <- c("Control", "Truncated", "Logarithmic")
+vals <- c("#1c9e77", "#d95f02", "#7570b3")
+
+ggplot() +
+  geom_density(data = as.data.frame(control), aes(x=control, col = "Control"))+
+  geom_density(data = as.data.frame(truncated), aes(x=truncated, col = "Truncated"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks[1:2], values = vals[1:2])+
+  theme_classic()
+
+```
+As expected from the above analysis the two distributions are very similar and we can see that they are, in fact, very much non-normal and asymmetrical. Similarly, the box plot of the response sets (below) displays the similarity between them, where the boxes look almost identical aside from the black square depicting the mean for the truncated plot being slightly higher than for the control.
```

Comment:
```suggestion
```

As expected from the above analysis the two distributions are similar and we can see that they are, in fact, non-normal and asymmetrical. Similarly, the box plot of the response sets (below) displays the similarity between them, where the boxes look almost identical aside from the black square depicting the mean for the truncated plot being slightly higher than for the control.
```